README.md

```diff
@@ -47,31 +47,31 @@ Installation
 
 There are various ways to install the firmware onto the device.
 Here we will cover using [OpenOCD](http://openocd.org/) with
-`gdb-multiarch`, using a Nucleo-F303ZE board to provide the ST-LINK
-interface.
+`gdb-multiarch`, using a Nucleo-F303ZE board or the STLINK-V3SET
+to provide the ST-LINK interface.
 
 ### Hardware setup
 
 1. Consult the
-   [UM1974](https://www.st.com/en/evaluation-tools/nucleo-f303ze.html#resource) datasheet.
-2. Locate the ST-LINK jumper pins on the Nucleo board and make sure they are
+   [UM1974](https://www.st.com/en/evaluation-tools/nucleo-f303ze.html#resource) datasheet. The pin connections for STLINK-V3 are similar, but with slightly different name.
+2. Locate the ST-LINK jumper pins on the programmer boards and make sure they are
    open.  See Figure 10: Using ST-LINK/V2-1 to program the STM32 on an
    external application in UM1974.
 3. Connect the SWD pins on the Nucleo board to the pins on the MPD
    bloodlight board.  The pins on the MPD bloodlight board are named on the
    board.  For the Nucleo board, check see Table 5. Debug connector CN6 (SWD).
    There is a small dot at one end of the SWD connector which marks pin 1.
 
-   | Nucleo board | MPD board |
-   | ------------- | --------- |
-   | VDD_TARGET    | 3.3V      |
-   | SWCLK         | SWCLK     |
-   | GND           | GND       |
-   | SWDIO         | SWDIO     |
-   | NRST          | NRST      |
-   | SWO           | SWO       |
+   | Nucleo board  | STLINK-V3 | MPD board |
+   | ------------- | --------- | --------- |
+   | VDD_TARGET    | T_VCC     | 3.3V      |
+   | SWCLK         | CLK       | SWCLK     |
+   | GND           | GND       | GND       |
+   | SWDIO         | DIO       | SWDIO     |
+   | NRST          | NRST      | NRST      |
+   | SWO           | SWO       | SWO       |
 
-4. Connect the Micro-USB port at the ST-LINK end of the Nucleo board
+4. Connect both the Micro-USB ports at the ST-LINK end and MPD board
```

Comment:
I'd change this to say:
4. Connect both the programmer and bloodlight boards to your computer using micro-USB cables as both require power to function. The order in which they're connected doesn't matter.

Reply:
+1